sycl/source/device_selector.cpp

```diff
@@ -46,6 +46,10 @@ device device_selector::select_device() const {
                 << "  device: " << DeviceName << std::endl;
     }
 
+    // A negative score means that a device must not be selected.
```

Comment:
I think https://github.com/intel/llvm/pull/1689 addresses the same issue. Could you take a look, please?

Reply:
I think it does, but the test on this PR is still relevant